src/common/snippets/src/utils/utils.cpp

```diff
@@ -199,11 +199,11 @@ VectorDims get_preordered_vdims(const VectorDims& shape, const std::vector<size_
 }
 
 VectorDims get_planar_vdims(const snippets::lowered::ExpressionPort& expr_port) {
-    OPENVINO_ASSERT(expr_port.get_type() == snippets::lowered::ExpressionPort::Type::Input, "get_planar_vdims expects Expression Input port");
+    assert(expr_port.get_type() == snippets::lowered::ExpressionPort::Type::Input && "get_planar_vdims expects Expression Input port");
```

Comment:
I think there should be `OPENVINO_ASSERT` too - by the same reason 🤔

Reply:
Done